conda-recipes/kfac-feedstock/.ci_support/linux_noarch.yaml

```diff
@@ -0,0 +1,16 @@
+c_compiler:
+- gcc
+c_compiler_version:
+- '8'
+channel_sources:
+- conda-forge,defaults
```

Comment:
I think this doesn't need conda-forge here.

Also, I think you can  put the WML CE channel here as well (first) - just the full URL like:

```
channel_sources:
  - https://public.dhe.ibm.com/ibmdl/export/pub/software/server/ibm-ai/conda/,defaults
```

Reply:
This should help the "clobber" warnings in the test part of the build.